### DISCUSSION TITLE
Disable default SignIn

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3429] [Help]
Hi, I would like to know is it possible to disable default SignIn section and set the default to 3rd party authentication provider as shown in the image below

![image](https://user-images.githubusercontent.com/597759/142353446-e4e0e67a-730e-4cef-bcfb-e21a99148901.png)

**Comment by mjashanks** [comment]
Hi there - this is not possible at the moment - though has been raised before: https://github.com/Budibase/budibase/discussions/2609

Give that one an upvote!
